### ISSUE TITLE
Package glib-2.0 was not found

### ISSUE THREAD

**Issue by ianchanning** [vhakulinen/gnvim#74]
My setup:

* Ubuntu 18.04
* Rustc 1.32.0

I get the following error when running `make`:

```sh
channi16@ICT0115123:~/v/gnvim$ make
cargo build --release
   Compiling onig_sys v69.1.0
   Compiling miniz-sys v0.1.12
   Compiling glib-sys v0.9.0
   Compiling byteorder v1.3.2
   Compiling log v0.4.7
   Compiling crc32fast v1.2.0
   Compiling idna v0.1.5
   Compiling rand_chacha v0.1.1
error: failed to run custom build command for `glib-sys v0.9.0`
process didn't exit successfully: `/var/www/vhosts/gnvim/target/release/build/glib-sys-a5c428d969f48a9a/build-script-build` (exit code: 1)
--- stderr
`"pkg-config" "--libs" "--cflags" "glib-2.0 >= 2.42"` did not exit successfully: exit code: 1
--- stderr
Package glib-2.0 was not found in the pkg-config search path.
Perhaps you should add the directory containing `glib-2.0.pc'
to the PKG_CONFIG_PATH environment variable
No package 'glib-2.0' found


warning: build failed, waiting for other jobs to finish...
error: build failed
Makefile:6: recipe for target 'build' failed
make: *** [build] Error 101
```

One of the [requirements for GTk-rs](https://gtk-rs.org/docs-src/requirements.html) for Fedora is that glib2 is must be installed, so it looks like I don't have glib2 installed either even though I'm on Ubuntu.

**Comment by ianchanning** [comment]
I fixed this problem with `sudo apt-get install libglib2.0-dev` (via [askubuntu answer](https://askubuntu.com/a/58357/8989), but I'm now hitting the next missing file:

```sh
channi16@ICT0115123:~/v/gnvim$ make
cargo build --release
   Compiling glib-sys v0.9.0
   Compiling miniz-sys v0.1.12
   Compiling onig_sys v69.1.0
   Compiling rand_os v0.1.3
   Compiling atty v0.2.12
   Compiling javascriptcore-rs-sys v0.2.0
   Compiling unix_socket v0.5.0
   Compiling rand_pcg v0.1.2
   Compiling num-traits v0.2.8
   Compiling quote v0.6.13
   Compiling pulldown-cmark v0.2.0
   Compiling serde_json v1.0.40
   Compiling serde_bytes v0.10.5
   Compiling base64 v0.10.1
   Compiling bincode v1.1.4
   Compiling url v1.7.2
   Compiling gobject-sys v0.9.0
   Compiling cairo-sys-rs v0.9.0
error: failed to run custom build command for `cairo-sys-rs v0.9.0`
process didn't exit successfully: `/var/www/vhosts/gnvim/target/release/build/cairo-sys-rs-c5c44a71aefb5374/build-script-build` (exit code: 1)
--- stderr
`"pkg-config" "--libs" "--cflags" "cairo-gobject >= 1.12"` did not exit successfully: exit code: 1
--- stderr
Package cairo-gobject was not found in the pkg-config search path.
Perhaps you should add the directory containing `cairo-gobject.pc'
to the PKG_CONFIG_PATH environment variable
No package 'cairo-gobject' found


warning: build failed, waiting for other jobs to finish...
error: build failed
Makefile:6: recipe for target 'build' failed
make: *** [build] Error 101
```

**Comment by ianchanning** [comment]
So it looks like I needed to install the full Gtk v2 dev package `sudo apt-get install libgtk2.0-dev`. This doesn't make much sense as the minimum required Gtk version is 3.18 and I had run the following command:

```sh
sudo apt install libgtk-3-dev libwebkit2gtk-4.0-dev
```

Hmm, I've now re-run that command. It looks like there were problems with my `apt-get` not finding some resources, I just needed to run `sudo apt-get update`.
